### PR TITLE
Junit test for the search client by name

### PR DESCRIPTION
**Pull request by agyen**

### CODE REVIEW

File: src/test/java/io/turntabl/ClientRegisterTest.java [hunk 1]
```diff
@@ -13,7 +13,7 @@
     Client client3 = new Client(003, "Yaa Nuamah", "Dansoman", "+23355345572", "yaa@gmail.com", ClientLevel.PREMIUM);
     Client client4 = new Client(004, "Sam Bartels", "Ashaiman", "+23355554572", "sam@gmail.com", ClientLevel.GOLD);
     Client client5 = new Client(005, "Sam Moorehouse", "Dansoma", "+233553458272", "Moore@gmail.com", ClientLevel.PLATINUM);
-    Client client6 = new Client(006, "James Barsely", "London", "+233553454472", "james96@gmail.com", ClientLevel.PREMIUM);
+    Client client6 = new Client(006, "Isaac Barsely", "London", "+233553454472", "james96@gmail.com", ClientLevel.PREMIUM);
```
Comment (Ankomahene) [inline src/test/java/io/turntabl/ClientRegisterTest.java:16]: Why did you change James to Isaac?

File: src/test/java/io/turntabl/ClientRegisterTest.java [hunk 2]
```diff
@@ -72,11 +72,42 @@ public void geClientByLevel_Platinum_NotPlatinumClients(){
 
     // Test for searching client by ID
     @Test
-    public void getClientById_IdEqualsClientsId(){
+    public void testGetClientById_IdEqualsClientsId(){
```
Comment (Ankomahene) [inline src/test/java/io/turntabl/ClientRegisterTest.java:75]: that's okay

File: src/test/java/io/turntabl/ClientRegisterTest.java (hunk 1 above)
Comment (agyen) [reply src/test/java/io/turntabl/ClientRegisterTest.java:16]: I used it for a unit test, thanks for asking mate :)

File: src/test/java/io/turntabl/ClientRegisterTest.java (hunk 2 above)
Comment (agyen) [reply src/test/java/io/turntabl/ClientRegisterTest.java:75]: cheers

File: src/main/java/io/turntabl/ClientRegister.java [hunk 3]
```diff
@@ -19,7 +19,7 @@ public ClientRegister(List<Client> clients) {
 
     public List<Client> getClientsByName(String name){
         return clients.stream()
-                .filter(client -> client.getName().equals(name))
+                .filter(client -> client.getName().toLowerCase().contains(name.toLowerCase()))
```
Comment (YaaNuamah) [inline src/main/java/io/turntabl/ClientRegister.java:22]: Your test code LGTM :+1:

File: src/test/java/io/turntabl/ClientRegisterTest.java (hunk 1 above)
Comment (Ankomahene) [reply src/test/java/io/turntabl/ClientRegisterTest.java:16]: Okay!
That's fine. LGTM

Comment (YaaNuamah) [inline src/test/java/io/turntabl/ClientRegisterTest.java:16]: Please could you explain why the name change ?

File: src/main/java/io/turntabl/ClientRegister.java (hunk 3 above)
Comment (agyen) [reply src/main/java/io/turntabl/ClientRegister.java:22]: thanks mate :)

File: src/test/java/io/turntabl/ClientRegisterTest.java (hunk 1 above)
Comment (agyen) [reply src/test/java/io/turntabl/ClientRegisterTest.java:16]: I used it for a unit test, thanks for asking mate :)

**Review by YaaNuamah — COMMENTED** [review]
Please I will be waiting for your feedback.

File: src/test/java/io/turntabl/ClientRegisterTest.java (hunk 1 above)
Comment (YaaNuamah) [reply src/test/java/io/turntabl/ClientRegisterTest.java:16]: Alright that's awesome.

Comment (YaaNuamah) [reply src/test/java/io/turntabl/ClientRegisterTest.java:16]: Okay works that is good to know.

**Comment by YaaNuamah** [comment]
LGTM
